rocAL/rocAL_pybind/examples/rocAL_api_video_pipeline.py

```diff
@@ -152,17 +160,11 @@ def main():
                 print("**************ends*******************")
                 print("**************", i, "*******************")
         data_loader.reset()
-    #Your statements here
+    # Your statements here
     stop = timeit.default_timer()
 
     print('\n Time: ', stop - start)
 
 
 if __name__ == '__main__':
     main()
```

Comment:
Introduce an EOF line

Reply:
Please address this comment

Reply:
It shows an empty line at EOF. I'll add another line and check